watcher/watcher.go

```diff
@@ -111,14 +105,14 @@ func (wh *WatchHandler) GetImageIDsForSBOMCalculation() ([]string, error) {
 }
 
 func (wh *WatchHandler) GetWlidsForImageID(imageID string) []string {
-	return wh.imagesIDToWlidsToContainerToImageIDMap[imageID]
+	return wh.imagesIDToWlidsMap[imageID]
```

Comment:
1. even though you are not changing it, use `imageIDsMapMutex`
2. what happens when the imageID is missing missing from the map?